packages/xo-web/src/common/intl/messages.js

```diff
@@ -754,6 +754,14 @@ const messages = {
   hostStatus: 'Status',
   hostBuildNumber: 'Build number',
   hostIscsiName: 'iSCSI name',
+  hostMultipathing: 'Multipathing',
+  hostEnableMultipathing: 'Enable multipathing',
+  hostDisableMultipathing: 'Disable multipathing',
+  hostEnableMultipathingForAllHosts: 'Enable all hosts multipathing',
+  hostDisableMultipathingForAllHosts: 'Disable all hosts multipathing',
```

Comment:
Why `host-` when it's actually used in the pool view? And let's try not to compel those message IDs to one unique use as much as possible.